packages/codefixes/src/typescript-codefix-collection.ts

```diff
@@ -16,7 +16,52 @@ import type { CodeFixCollection, DiagnosticWithContext } from './types';
 export class TypescriptCodeFixCollection implements CodeFixCollection {
   hasPrettier: boolean | undefined;
   prettierConfigs: PrettierOptions | undefined;
-  getFixForDiagnostic(diagnostic: DiagnosticWithContext): CodeFixAction | undefined {
+
+  // List of safe to apply code fix names
+  // @see ./helpers/available-fix-names.ts for the list of available fixNames
+  safeCodeFixNames = [
```

Comment:
We should move this to a separate file so we can eventually import a given set of `safeCodeFixNames` based on the CLI argument (for future API)

Reply:
eg https://github.com/rehearsal-js/rehearsal-js/issues/593

Reply:
P. S. I keep this list in the provider because different providers have different fixNames. I tried to keep it in the plugin, but it become maintenance more complicated.